servicedirectory/snippets.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import servicedirectory_v1beta1
+
+
+def create_namespace(project_id, location_id, namespace_id):
+  """Creates a namespace in the given location."""
+
+  client = servicedirectory_v1beta1.RegistrationServiceClient()
+
+  namespace = servicedirectory_v1beta1.Namespace(
+      name='projects/{0}/locations/{1}/namespaces/{2}'.format(
```

Comment:
You can use the path resource helper methods for constructing some of these strings (namespace, endpoint, and parent) https://googleapis.dev/python/servicedirectory/latest/servicedirectory_v1beta1/services.html#google.cloud.servicedirectory_v1beta1.RegistrationServiceClient.namespace_path